Lib/test/test_genericalias.py

```diff
@@ -394,6 +396,7 @@ def test_pickle(self):
                     self.assertEqual(loaded.__origin__, alias.__origin__)
                     self.assertEqual(loaded.__args__, alias.__args__)
                     self.assertEqual(loaded.__parameters__, alias.__parameters__)
+                    self.assertEqual(type(loaded), type(alias))
```

Comment:
maybe assertIs?

Reply:
It is the same for types. And I do not want to restart the CI testing for such minor change. :wink: